Tests/NIOTests/ByteBufferLengthPrefixTests+XCTest.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 0 Apple Inc. and the SwiftNIO project authors
```

Comment:
Something's gone wrong here with the year... was it generated like this?

Reply:
#1903.

Reply:
Good Catch, thanks. Updated to 2021